airflow/providers/databricks/hooks/databricks.py

```diff
@@ -356,31 +353,31 @@ def _do_api_call(self, endpoint_info, json):
     def _log_request_error(self, attempt_num: int, error: str) -> None:
         self.log.error('Attempt %s API Request to Databricks failed with reason: %s', attempt_num, error)
 
-    def run_now(self, json: dict) -> str:
+    def run_now(self, json: dict) -> int:
```

Comment:
can it break existing code? for example if people are using this result to concatenate with log string without using `str`?

Reply:
It won't break existing code, actually it's backwards - if someone assumes that output is `str` because of the function signature, then it'd break the code, because the actual returned type is `int`.